src/Ad/AbstractAdNode.php

```diff
@@ -81,15 +81,15 @@ public function setSequence($sequence)
         $this->domElement->setAttribute('sequence', $sequence);
         return $this;
     }
-    
+
     /**
-     * @return int
+     * @return string
```

Comment:
```
sequence: a number greater than zero (0) that identifies the sequence in which an ad should play; all
<Ad> elements with sequence values are part of a pod and are intended to be played in sequence
```
Why you change it ti string?

Reply:
You can refer the second item in Changed log.

Reply:
not sure i understand you